github/githubapp.go

```diff
@@ -0,0 +1,109 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+	"strconv"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+type GitHubApp struct {
+	ClientId       string `json:"client_id"`
+	InstallationId int64  `json:"installation_id"`
+	PrivateKey     string `json:"private_key"` // SENSITIVE
+}
+
+func (app *GitHubApp) IsZero() bool {
```

Comment:
Mhh. The logic in IsZero will return true if the struct is half constructed for any reason; this could hide a bug.
Was there a specific reason for this logic? Otherwise you could just compare with the zero value of the type, as it was in the previous version:

untested:
```
return *app == GitHubApp{}
```

Reply:
Fixed... + added tests